rllib/core/models/catalog.py

```diff
@@ -224,3 +240,80 @@ def get_tokenizer_config(
                 **{"use_lstm": False, "use_attention": False},
             },
         )
+
+    @classmethod
+    def get_action_dist_cls_dict(
+        cls,
+        action_space: gym.Space,
+        model_config_dict: ModelConfigDict,
```

Comment:
I don't see how you use `model_config_dict` here. Is this currently used for inferring the dist classes? I think for customization it's better to have the user override this function of the catalog and return their custom dist class with a custom catalog instead of passing it `model_config_dict`

Reply:
Yeah absolutely.

Reply:
The only issue is that it is needed for the `required_model_output_shape` methods that we need to compute input shapes for multi-distributions (that are not here yet).

Reply:
`get_dist_class` won't need model_config but `get_dist_object` will, right?